### PR TITLE
Update configure-sharepoint-content-source.md

### PR DESCRIPTION
**Pull request by AlexanderHolmeset**
The viva learning content provider menu is moved from admin.microsoft.com in to the admin tab of the Viva Learning app in Teams. Viva learning app in teams also links back to this outdated doc.

### CODE REVIEW

File: Viva/learning/configure-sharepoint-content-source.md [hunk 1]
```diff
@@ -67,27 +67,29 @@ The Learning Service uses the provided folder URLs to get metadata from all cont
 
 To configure SharePoint as a learning content sources in for Viva Learning, follow these steps:
 
-1. In the left navigation of the Microsoft 365 admin center, go to **Settings** > **Org settings**.
+1. Open Viva Learning in Microsoft Teams, go to **Admin** > **Manage Providers**.
```
Comment (JohanFreelancer9) [inline Viva/learning/configure-sharepoint-content-source.md:70]:
```suggestion
1. Open Viva Learning in Microsoft Teams and go to **Admin** > **Manage Providers**.
```

File: Viva/learning/configure-sharepoint-content-source.md [hunk 2]
```diff
@@ -67,27 +67,29 @@ The Learning Service uses the provided folder URLs to get metadata from all cont
...
+<img width="1510" alt="image" src="https://github.com/MicrosoftDocs/Viva/assets/86094924/cab8c569-39b2-4880-9096-1cab51f2b9b3">
 
-2. On the **Org settings** page, on the **Services** tab, select **Viva Learning**.
 
-     ![Settings page in the Microsoft 365 admin center showing Viva Learning listed.](../media/learning/clcs-services.png)
+3. Click Add Provider, select Share Point and click Next.
```
Comment (JohanFreelancer9) [inline Viva/learning/configure-sharepoint-content-source.md:74]:
```suggestion
3. Select **Add Provider** > **Share Point** > **Next**.
```

File: Viva/learning/configure-sharepoint-content-source.md [hunk 3]
```diff
@@ -67,27 +67,29 @@ The Learning Service uses the provided folder URLs to get metadata from all cont
...
+<img width="605" alt="image" src="https://github.com/MicrosoftDocs/Viva/assets/86094924/dc1b7523-c7e2-4f88-9473-28f6dcdf39f3">
 
-3. On the **Viva Learning** panel, under SharePoint, provide the site URL to the SharePoint site where you want Viva Learning to create a centralized repository. If your SharePoint site is new, you'll need to wait 1 hour after site creation to add it here. You must also be the owner of the SharePoint site.
+
+4. Set a Display name, provide the site URL to the SharePoint site where you want Viva Learning to create a centralized repository, click Save. If your SharePoint site is new, you'll need to wait 1 hour after site creation to add it here. You must also be the owner of the SharePoint site.
```
Comment (JohanFreelancer9) [inline Viva/learning/configure-sharepoint-content-source.md:78]:
```suggestion
4. Set a display name, provide the site URL to the SharePoint site where you want Viva Learning to create a centralized repository, and select **Save**. If your SharePoint site is new, you'll need to wait one hour after site creation to add it here. You must also be the owner of the SharePoint site.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AlexanderHolmeset, kindly commit the suggested changes. Many thanks.

**Comment by AlexanderHolmeset** [comment]
Looks good, thanks for the quick feedback

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@bhaswatic 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**